02/app.js

```diff
@@ -1 +1,12 @@
-console.log('DOM');
\ No newline at end of file
+console.log("DOM");
+
+const dataUrlList = document.querySelectorAll("a");
+if (dataUrlList) {

```

Comment:
Jeśli wyszukujesz wiele elementów i żaden nie został odnaleziony to jest zwracana pusta tablica. Pusta tablica jest obiektem więc jest prawdą. Dlatego waki warunke nie ma sensu bo będzie zawsze prawdziwy. Można go pominać lub poprawić na coś takiego: `if (dataUrlList.length > 0)`